cardano-crypto-tests/src/Test/Crypto/Regressions.hs

```diff
@@ -0,0 +1,29 @@
+{-# LANGUAGE CPP #-}
+{-# LANGUAGE OverloadedStrings #-}
+#ifdef SECP256K1_ENABLED
+{-# LANGUAGE TypeApplications #-}
```

Comment:
Why do we need to dance around `TypeApplications`?

Reply:
I get warnings otherwise.